src/NHibernate/Dialect/SybaseSQLAnywhere10Dialect.cs

```diff
@@ -140,12 +140,13 @@ protected virtual void RegisterMathFunctions()
 			RegisterFunction("radians", new StandardSQLFunction("radians", NHibernateUtil.Double));
 			RegisterFunction("rand", new StandardSQLFunction("rand", NHibernateUtil.Double));
 			RegisterFunction("remainder", new StandardSQLFunction("remainder"));
-			RegisterFunction("round", new StandardSQLFunction("round"));
+			RegisterFunction("round", new RoundEmulatingSingleParameterFunction("round"));
```

Comment:
Tested with an Anywhere 17, using [this custom driver](https://github.com/fredericDelaporte/nhibernate-core/blob/Anywhere17/src/NHibernate/Driver/SapSQLAnywhere17Driver.cs).